### PR TITLE
Improve: tweak sample project

### PR DESCRIPTION
**Pull request by 3lvis**
- Increases amount of sections
- Increases the height of the section headers and centers their labels
- Remove local photos related code
- Refactors titleForSection to have a more Swift 3 kind of API
- Remove additional newlines to make code formatting more consistent
- Remove attribution header from UILabel extension

d

### CODE REVIEW

File: Sources/DateScrubber.swift [hunk 1]
```diff
@@ -1,20 +1,16 @@
 import UIKit
 
 public protocol DateScrubberDelegate {
-
```
Comment (marijnschilling) [inline Sources/DateScrubber.swift:4]: Right! Less white space, i get it :v:

File: iOS/RemoteCollectionController.swift [hunk 2]
```diff
@@ -80,24 +71,20 @@ extension RemoteCollectionController {
...
        
         return headerView
     }
-
 }
 
-extension RemoteCollectionController{
-
+extension RemoteCollectionController: DateScrubberDelegate {
     override func scrollViewDidScroll(scrollView: UIScrollView){
```
Comment (marijnschilling) [inline iOS/RemoteCollectionController.swift:82]: Hmmm yeah, i guess it's nice to implement the protocol here

**Comment by 3lvis** [comment]
Really solid implementation @marijnschilling 👏 We might need to consider to rename this to be more generic since it can work with anything.

**Comment by marijnschilling** [comment]
👍  Merge it!

alt text

**Comment by marijnschilling** [comment]
Renaming would be nice, did realise calling it a dateScrubber when you're just feeding it Strings is a bit weird.

**Comment by 3lvis** [comment]
pumpgirl

**Pull request merged**